src/views/tutorial-view/components/featured-in-collections/helpers.ts

```diff
@@ -1,15 +1,39 @@
-import { Collection as ClientCollection } from 'lib/learn-client/types'
-import { CollectionCardProps } from '../collection-card'
+import { getCollectionSlug } from 'views/collection-view/helpers'
+import {
+  Collection as ClientCollection,
+  themeIsProduct,
+} from 'lib/learn-client/types'
+import { ProductSlug } from 'types/products'
+import { CollectionCardPropsWithId } from './types'
 
 export function formatCollectionCard(
   collection: ClientCollection
-): CollectionCardProps {
+): CollectionCardPropsWithId {
   return {
     id: collection.id,
-    name: collection.name,
-    slug: collection.slug,
     description: collection.description,
-    theme: collection.theme,
-    numTutorials: collection.tutorials.length,
+    heading: collection.name,
+    productsUsed: parseProductsUsed(collection),
+    tutorialCount: collection.tutorials.length,
+    url: getCollectionSlug(collection.slug),
+  }
+}
+
+/**
+ * TODO: Designs show multiple productsUsed on collection cards.
+ *
+ * Should we parse the tutorials within a collection in order
+ * to show a more complete list? Or will this result in more of a
+ * "product icon overload" on these cards?
```

Comment:
🤔 Hmmmmm..... Currently on learn the collection card logos render their theme only.

Reply:
My hunch is to stick with current behavior, if we want to adjust cloud logos to also render all their tutorials product options, we can adjust later? 🤷‍♀️

Reply:
If we do that, we may want to rethink using the `productsUsed` prop, or just writing a note that this is actually just `theme`

Reply:
My aim with the implementation here is to match that behaviour! Added this note cause i'm seeing many product icons in the designs:

<img width="717" alt="CleanShot 2022-04-21 at 17 15 44@2x" src="https://user-images.githubusercontent.com/4624598/164553667-7eca29b3-cefd-415f-9e8f-8cc02e74dc21.png">

I think this is currently impossible? So just added this as a reminder that we should check in with design on behaviour here, at some point. Let me know if that make sense - if so I'll open & link to an Asana ticket in the comment, if not I'll drop the comment 👍

Reply:
ya that sounds good 😄

Reply:
Sorry I was just testing how many badges we could fit at each size here. I know there can only be one currently but I wanted to be flexible for the future as well!

Reply:
@braydenlove sweet, makes total sense!